lib/font_awesome_flutter.dart

```diff
@@ -1545,4 +1555,1401 @@ class FontAwesomeIcons {
   static const IconData youtube = const IconDataBrands(0xf167);
   static const IconData youtubeSquare = const IconDataBrands(0xf431);
   static const IconData zhihu = const IconDataBrands(0xf63f);
+
+  static const values = <IconData>[
```

Comment:
Could you describe your use case / reason for adding this? Couldn't quite think of why you'd want to iterate over all icons, haha :)

Reply:
I did this PR specifically for this reason.
My friend wants to display a colleciton of icons the user can choose from. There was no way to do this previously to my knowledge, so I added the `values` to easily iterate and display all icons.

Reply:
Ah, gotcha. Since this is the first time I've gotten this particular request, I'm wondering if it might be better to fork this library for that specific project? 

My only thought is that I don't want to add a pretty large array to the class if only 1 project is going to use it. Do you know if tree-shaking would remove this array from most people's production code? I get the feeling it would, but I never quite know.

Reply:
That's a good question. I don't know if it would be removed by tree-shaking. I'll close this and instead fork it for my purpose instead until I know for sure.